src/libprojplugin/libprojprojectnodes.cpp

```diff
@@ -22,6 +21,18 @@ OwnProjectNode::OwnProjectNode(OwnProject * Project, OwnProjectFile * ProjectFil
     setDisplayName(projectFile->filePath().toFileInfo().completeBaseName());
 }
 
+QList<ProjectExplorer::ProjectAction> OwnProjectNode::supportedActions(Node *node) const
+{
+    qDebug() << "Calling OwnProjectNode::supportedActions(Node *node) const";
+    Q_UNUSED(node);
+    return QList<ProjectAction>()
+        << ProjectExplorer::ProjectAction::AddNewFile
```

Comment:
Try to don't disable code via comments. Just remove it, you already use version control.